circle.yml

```diff
@@ -51,7 +51,8 @@ dependencies:
   override:
     - .circle/docker-compose.sh pull ${DISTRO}
   post:
-    - .circle/docker-compose.sh build ${DISTRO}
+    - .circle/docker-compose.sh build_st2 ${DISTRO}
```

Comment:
See ec415de2d785c66857bc7de45d5973984a7e7802, https://github.com/StackStorm/st2/pull/2580/commits/48e8ac84e20014ec718e33ba62c96df056cf50bf and https://github.com/StackStorm/mistral/pull/14/commits/f3ff2ce353240acecfc7ede034ccf8861e557fc6